modules/adding-tab-pods-page.adoc

```diff
@@ -97,5 +98,18 @@ export default function ExampleTab() {
 }
 ----
 
+. Install a Helm chart with the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option to deploy your plugin on a cluster. Provide the location of the image within the `plugin.image` parameter by using the following command:
+
++
+[source,terminal]
+----
+$ helm upgrade -i  my-plugin charts/openshift-console-plugin -n my-plugin-namespace --create-namespace --set plugin.image=my-plugin-image-location
```

Comment:
this is a suggestion, but I think longer commands benefit from being broken out into multiple lines, e.g. maybe one option or parameter per line for readability:
```suggestion
$ helm upgrade -i  my-plugin charts/openshift-console-plugin \
  -n my-plugin-namespace \
  --create-namespace \
  --set plugin.image=my-plugin-image-location
```